### ISSUE TITLE
Time by azimuth not working

### ISSUE THREAD

**Issue by MikeyMikeDO** [rdmtc/node-red-contrib-sun-position#342]
**Describe the bug**
The function time by azimuth is´nt working. 
The output is completely wrong. "0" degree gives a time at the afternoon. Changing the degree causes an incomprehensible result. The tooltip of the inputfield shows an error:
 "Cannot read property 'ts' of undefined, kann den Wert von pdsTimeByAzimuth.0 nicht ermitteln!"

**Screenshots**
![Github_sunmoon1](https://user-images.githubusercontent.com/47368571/139388452-8c5e7aba-f7ea-40a3-bbd6-334fa604dd18.JPG)

![gibthub2_sundandmoon](https://user-images.githubusercontent.com/47368571/139389203-33f1b570-c8f9-4a50-bbc9-ddb0119d5ca1.jpg)

**Expected behavior**
To get a correct timestamp of a given anzimuth degree at the definded local postion.

**System information (please complete the following information):**
 - Node Version 1.2.4
 - Node-Red Version 2.0.4 + 2.1.3

**Additional context**
In an older thread i found there was another person with the same problemen bit without any solution.
[https://githubmemory.com/repo/rdmtc/node-red-contrib-sun-position/issues/231]

```json
[{"id":"d3c04d4e.ce3e3","type":"debug","z":"2f24705ea71ab96a","name":"","active":true,"tosidebar":true,"console":false,"tostatus":false,"complete":"true","x":450,"y":200,"wires":[]},{"id":"3e4d36c0.620e7a","type":"time-inject","z":"2f24705ea71ab96a","name":"","nameInt":"⏲ \"time by azimuth\" = Zeitpunkt↷","positionConfig":"8c7609ce.dd5798","props":[{"p":"","pt":"msgPayload","v":"","vt":"date","o":"1","oT":"num","oM":"60000","f":0,"fS":0,"fT":"Millisekunden UNIX-Zeit","fI":"0","next":true,"days":"","months":"","onlyOddDays":false,"onlyEvenDays":false},{"p":"","pt":"msgTopic","v":"","vt":"str","o":"1","oT":"none","oM":"60000","f":0,"fS":0,"fT":"Millisekunden UNIX-Zeit","fI":"0","next":false,"days":"","months":"","onlyOddDays":false,"onlyEvenDays":false}],"injectTypeSelect":"time","intervalCount":1,"intervalCountType":"num","intervalCountMultiplier":60000,"time":"0","timeType":"pdsTimeByAzimuth","offset":0,"offsetType":"none","offsetMultiplier":60000,"timeEnd":"","timeEndType":"entered","timeEndOffset":0,"timeEndOffsetType":"num","timeEndOffsetMultiplier":60000,"timeDays":"*","timeOnlyOddDays":false,"timeOnlyEvenDays":false,"timeMonths":"*","timedatestart":"","timedateend":"","property":"","propertyType":"none","propertyCompare":"true","propertyThreshold":"","propertyThresholdType":"num","timeAlt":"","timeAltType":"entered","timeAltDays":"*","timeAltOnlyOddDays":false,"timeAltOnlyEvenDays":false,"timeAltMonths":"*","timeAltOffset":0,"timeAltOffsetType":"num","timeAltOffsetMultiplier":60000,"once":false,"onceDelay":"","recalcTime":"","x":200,"y":200,"wires":[["d3c04d4e.ce3e3"]]},{"id":"8c7609ce.dd5798","type":"position-config","name":"Garten","isValide":"true","longitude":"0","latitude":"0","angleType":"deg","timeZoneOffset":"99","timeZoneDST":"0","stateTimeFormat":"3","stateDateFormat":"12"}]
```

**Comment by Hypnos3** [comment]
Hallo,

maybe this is already fixed in the node-red-contrib-sun-position@2.0.0-beta-14.

It will be a while before I can test this. However, you can already install and use the new version via the command line.

regards

**Comment by Hypnos3** [comment]
I have tested this with 2.0.0 and there the value seems to be correct:
![image](https://user-images.githubusercontent.com/12692680/140612007-5afa8d0f-af1b-429d-9915-6f7b73f98746.png)
